### ISSUE TITLE
Buddybuild crash report on build #252

### ISSUE THREAD

**Issue by sarentz** [mozilla-mobile/prox#172]
Buddybuild detected a crash from antlam@mozilla.com
`Prox/AppDelegate.swift` line 9`
```swift
import UIKit
import Firebase

@UIApplicationMain
class AppDelegate: UIResponder, UIApplicationDelegate { <-- CRASHED

    var window: UIWindow?
    var placeCarouselViewController: PlaceCarouselViewController?

    private var authorizedUser: FIRUser?
```

[View File](https://github.com/mozilla-mobile/prox/blob/4a8e7678bff29a418fef4e401b2e94494698140c/Prox/AppDelegate.swift#L9) | [View Full Crash Details](https://dashboard.buddybuild.com/apps/57f51f8bcaa70a01005412ee/crashreport/27b13f33f4104e50b74674421bd87158)

**Comment by mcomella** [comment]
Build 252 doesn't have my fix from #166.

**Comment by sarentz** [comment]
Crash report from sarentz@mozilla.com
